docs/Board - CC3D.md

```diff
@@ -103,28 +103,58 @@ You cannot use USART3 and I2C at the same time.
 
 # Flashing
 
-There are two primary ways to get Cleanflight onto a CC3D board.
+Since 1.11.0 release there is only one way to get Cleanflight onto a CC3D board: flash it to board using an FTDI USB-TTL converter and special software:
+* Windows only: (STM Flash Loader Demonstrator)[http://www.st.com/web/en/catalog/tools/PF257525]
+* Windows/Linux/OS X: (stm32flash utility)[http://sourceforge.net/projects/stm32flash/]. Here is a (manual online)[http://manpages.ubuntu.com/manpages/wily/man1/stm32flash.1.html]
 
-* Single binary image mode - best mode if you don't want to use OpenPilot.
-* OpenPilot Bootloader compatible image mode - best mode if you want to switch between OpenPilot and Cleanflight.
+Please not that some user have issues with flashing CC3D via non-FTDI adapter. In that case "STM Flash Loader Demonstrator" just cannot connect to the target. Also, Arduino's embedded FTDIs may not work too.
```

Comment:
Please not**e** that some user have issues with flashing CC3D via non-FTDI adapter. In that case "STM Flash Loader Demonstrator" just cannot connect to the target. ~~Also,~~ Arduino's embedded FTDIs may **also** not work ~~too~~.